src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/List.cshtml

```diff
@@ -24,93 +24,116 @@
     <input type="submit" name="submit.BulkAction" class="d-none" />
     <input asp-for="Options.BulkAction" type="hidden" />
 
-    <fieldset class="filter-options">
-        <div class="card mb-3">
-            <div class="card-body bg-light p-3">
+
+    @if (Model.ContentItems.Count > 0)
+    {
+        <ul class="list-group with-checkbox">
+            <li class="list-group-item bg-light">
                 <div class="row">
                     <div class="form-group col-xl-3 mb-xl-n1">
-                        <div class="input-group input-group-sm">
-                            <div class="input-group-prepend">
-                                <span class="input-group-text bg-secondary text-light">@T["Title"]</span>
-                            </div>
-                            <input asp-for="Options.DisplayText" class="form-control" />
+                        <div class="custom-control custom-checkbox mr-n2">
+                            <input type="checkbox" class="custom-control-input" id="select-all">
+                            <label class="custom-control-label" for="select-all" id="select-all-label" title="@T["Select All"]"></label>
+                            <label id="selected-items" class="text-muted" for="select-all">@T.Plural(Model.ContentItems.Count, "1 item", "{0} items")</label>
+                            <span class="dropdown">
+                                <button disabled class="btn btn-light btn-sm dropdown-toggle" type="button" id="bulk-action-menu-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
It would be nice if we make Bulk Action drop down like the other filters for consistency